cogs/rtfm.py

```diff
@@ -2,6 +2,7 @@
 import os
 import re
 import zlib
+import unicodedata
```

Comment:
Unused import

Reply:
Oh sorry forgot to remove it after moving the command to etc.py.